ext/openssl/openssl.c

```diff
@@ -1355,32 +1430,73 @@ static X509 * php_openssl_x509_from_zval(zval * val, int makeresource, zend_reso
 		return NULL;
 	}
 
-	if (makeresource && resourceval) {
-		*resourceval = zend_register_resource(cert, le_x509);
-	}
 	return cert;
 }
 
+/* {{{ php_openssl_x509_from_param
+	Given a parameter, extract it into an X509 object.
+	The parameter can be:
+		. X509 object created using openssl_read_x509()
+		. if it starts with file:// then it will be interpreted as the path to that cert
+		. it will be interpreted as the cert data
```

Comment:
it's still handled here https://github.com/php/php-src/pull/5860/files#diff-69bad938d17f4283faa5f7fea17fa627R1412 but not too visible from the diff (I had to open the file to see it :) )

Reply:
To be clear, what I was suggesting here is that a word is missing in the comment. Or maybe more than just a word, as "The parameter can be: it will be interpreted as the cert data" does not make a whole lot of sense.

Reply:
yeah this was just copied from previous description. It should be probably something like `if string (or object convertible to string) it will be interpreted as PEM`